packages/config/config/devices/0x010f/fgwdeu.json

```diff
@@ -272,45 +262,23 @@
 		},
 		"40": {
 			"label": "First button – scenes sent",
-			"description": "determines which actions result in sending scene IDs",
+			"description": "determines which actions result in sending scene IDs (0 - no scenes; 1 - Key pressed once; 2 - Key pressed twice; 4 - Key pressed 3 times; 8 - Key held down and released)",
```

Comment:
This should be modeled as [partial parameters](https://zwave-js.github.io/node-zwave-js/#/config-files/file-format?id=partial-parameters)